RelhaxModpack/RelhaxModpack/Database/DownloadInstructions.cs

```diff
@@ -1,17 +1,7 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+﻿using RelhaxModpack.Utilities.Enums;
```

Comment:
I see there was some cleaning of not used `using`s. I meant to do that. Eventually.